tests/conftest.py

```
@@ -0,0 +1,14 @@
+import pytest
+
+
+@pytest.fixture(autouse=True)
+def mock_streamlit_fragment(monkeypatch):
+    """Globally bypass st.fragment decorator for coverage.
+
+    Ensures decorated functions are counted in functional coverage reports.
+    """
+
+    def identity(func):
```

Ruff (0.15.12)

[warning] Missing return type annotation for private function `identity`

(ANN202)